devtools-ci-index.yaml

```diff
@@ -1177,7 +1133,6 @@
             git_repo: ngx-fabric8-wit
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_build_deploy.sh'
-            svc_name: fabric8-ui-ngx-fabric8-wit
```

Comment:
Because for fabric8-ui we need to use saasherder and this service never existed and never worked (see https://ci.centos.org/view/Devtools/job/devtools-ngx-fabric8-wit-npm-publish-build-master/98/console from March or https://ci.centos.org/view/Devtools/job/devtools-ngx-fabric8-wit-npm-publish-build-master/125/console from April)